config/paths-fedora.conf

```diff
@@ -34,7 +34,7 @@ apache_access_log = /var/log/httpd/*access_log
 
 exim_main_log = /var/log/exim/main.log
 
-mysql_log = /var/lib/mysql/mysqld.log
+mysql_log = /var/log/mariadb/mariadb.log
```

Comment:
what's about both? I've not tried it, but think it should be possible in includes also, since we support multiline there... I mean, something like:
```
mysql_log = /var/log/mysqld.log
            /var/log/mariadb/mariadb.log
```